tapiriik/sync/sync.py

```diff
@@ -205,7 +205,10 @@ def PerformGlobalSync(self, heartbeat_callback=None, version=None):
                     body_dict = json.loads(message.body)
                     receipt_handle = message.receipt_handle
                     #print('[Sync.PerformGlobalSync]--- Sync user : {0}'.format(body_dict['user_id']))
-                    _global_logger.info('Sync user : {0}'.format(body_dict['user_id']))
+                    if body_dict.get("user_id") is None:

```

Comment:
Added security check even if it worked great without it before (No occurence of missing 'user_id' key tracebacks in logs)

Reply:
je comprends pas trop, tu enlèves le log et tu mets un if mais il n'y a rien dedans non ?